continuationsPlugin/src/main/scala/continuations/DefDefTransforms.scala

```diff
@@ -843,6 +847,64 @@ object DefDefTransforms extends TreesChecks:
           )
       )
 
+      val createSymbol =
+        newSymbol(
+          continuationsStateMachineSymbol,
+          Names.termName("create"),
+          Flags.Override | Flags.Method,
+          MethodType(
+            List(termName("value"), termName("completion")),
+            List(anyOrNullType, continuationClassRef.appliedTo(anyOrNullType)),
+            continuationClassRef.appliedTo(defn.UnitType)
```

Comment:
Could we change the overridden return type of `create` to be `BaseContinuationImpl`, so that the `asInstanceOf` below would not be needed?